scss/modules/_footer.scss

```diff
@@ -21,31 +21,49 @@
   background-color: $navy;
   color: $inverse;
   font-family: $sans-serif;
-  padding: u(1rem 0);
+  padding: u(2rem 0 1rem 0);
 
   a {
-    border-bottom-color: $inverse;
+    border-bottom: none;
     color: $inverse;
+
+    &:hover {
+      border-bottom: 1px dotted $inverse;
+    }
   }
 
-  .footer-links__column li {
-    font-size: u(1.8rem);
+  li {
+    font-size: u(1.6rem);
+    padding: (.5rem 0);
```

Comment:
Good idea. How does this look to you?
<img width="768" alt="screen shot 2017-03-22 at 10 05 19 pm" src="https://cloud.githubusercontent.com/assets/24054/24231735/eeb3c96e-0f4b-11e7-8e84-3f9cad27ace9.png">